### ISSUE TITLE
Some keywords

### ISSUE THREAD

**Issue by BifrostLuv** [BifrostLuv/Hawaii#4]
####...??...####
1.关系代数(Relational algebra)
2.笛卡尔积(Cartesian product)
3.代数与数理逻辑(Algebraic logic)

####...ML...####
1.回归分析 Regression:
简单线性回归 Simple Linear Regression
多元线性回归 Multiple Linear Regression
多项式回归 Polynomial Regression
支持向量机回归 Support Vector Regression
决策树 Decision Trees
随机森林 Random Forests
评估模型表现 ：R-Squared

2.分类分析 Classification:
逻辑回归 Logistic Regression
k近邻 K-Nearest Neighbors (K-NN)
支持向量机 Support Vector Machine (SVM)
朴素贝叶斯 Naive Bayes
决策树 Decision Trees
随机森林 Random Forests
梯度提升决策树 Gradient Boosted Decision Trees (XGBoost: 新兴算法)
评估模型表现：Confusion Matrix、Precision-Recall、ROC/AUC，etc

3.模型选择 Model Selection :
Cross Validation
Grid Search

4.聚类分析 Clustering :
K-Means Clustering
Hierarchical Clustering

5.深度学习 Deep Learning :
人工神经网络 Artificial Neural Network (ANN)
（卷积神经网络 Convolutional Neural Network (CNN)：图像处理）

6.降维 Dimensionality Reduction :
主成分分析 Principal Component Analysis (PCA)
(Linear Discriminant Analysis)
(Kernel PCA)
